frontend-ssr/redux/lesson-comments/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface CommentState {
```

Comment:
Just realized this comments state is not ok. With the current state we only save the comments of one lesson at a time. 

Ideally we would persist the comments, so that if you jump between lessons they are still there.  So the comments need to be related to a specific lesson - through a `lessonId`.